src/tilepy/include/PointingTools.py

```diff
@@ -1727,11 +1870,25 @@ def ComputeProbability2D(prob, highres, radecs, reducedNside, HRnside, minProbcu
             separations = skycoord.separation(MaxCoord)
             tempmask = separations < (radius + 0.05 * radius) * u.deg
             tempmask2 = separations > (radius - 0.05 * radius) * u.deg
+            hp.visufunc.projplot(skycoord[tempmask & tempmask2].ra, skycoord[tempmask &
+                                 tempmask2].dec, 'r.', lonlat=True, coord="C", linewidth=0.1)
             hp.visufunc.projplot(skycoord[tempmask & tempmask2].ra, skycoord[tempmask &
                                  tempmask2].dec, 'r.', lonlat=True, coord="C", linewidth=0.1)
             altcoord = np.empty(1000)
             azcoord = np.random.rand(1000) * 360
 
+            if ipixlistOcc == None:
+                print("No pcculted pix")
```

Comment:
typo. Also there that if it is just for testing. In general you would like to not have a print and instead only have:
if ipixlistOcc!=None:

Reply:
fixed